conans/model/env.py

```diff
@@ -0,0 +1,184 @@
+from collections import OrderedDict
+from conans.util.log import logger
+import os
+import re
+
+
+class EnvValues(object):
+    """ Object to represent the introduced env values entered by the user
+    with the -e or profiles etc.
+        self._data is a dictionary with: {(package, var): value}. "package" can be None if the var is global.
```

Comment:
Ok, I can try it, not sure if there is a good reason, just decided to do that way.